### PR TITLE
UI screens & UserName Edit Feature

### PR DESCRIPTION
**Pull request by HarshithaKonduru214**
Hey,please review
Thank you
chrome-capture (3)
chrome-capture (4)

### CODE REVIEW

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import "./home.css";
+
+export default function Home() {
+  const focusInput = useRef("");
+  const [focusText, setFocusText] = useState("");
+  const [focusState, setFocusState] = useState(false);
+  const [checked, setChecked] = useState(false);
+
+  const [logoutMenu, setLogoutMenu] = useState(false);
+  const [todoDisplay, setTodoDisplay] = useState(false);
```
Comment (rohanmathur91) [inline src/pages/Home/Home.jsx:19]: suggestion: try to refactor it and use `useReducer` as you have more than 3 states

File: src/pages/Welcome/Welcome.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          setName(nameInput.current.value);
+        }}
+      />
+
+      {name !== "" ? (
+        <button
+          className="welcome-button"
+          onClick={() => {
+            localStorage.setItem("userName", nameInput.current.value);
+            window.location.reload(false);
```
Comment (rohanmathur91) [inline src/pages/Welcome/Welcome.jsx:27]: can you explain what this is doing ?

**Review by rohanmathur91 — APPROVED** [review]
- Liked the UI of the extension
- Code is also well written and easy to understand
- Suggestion you can put `reset.css` in the styles folder

File: src/pages/Welcome/Welcome.jsx (hunk 2 above)
Comment (HarshithaKonduru214) [reply src/pages/Welcome/Welcome.jsx:27]: Basically, for the extension to be hosted on a web store it cannot have routes
So here it's setting the user name in local storage and triggering a reload to read that state and render the appropriate page ( the main page if the name is already set or the welcome page if the user still hasn't been set)

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (HarshithaKonduru214) [reply src/pages/Home/Home.jsx:19]: will do that, thanks!

File: src/pages/Welcome/Welcome.jsx (hunk 2 above)
Comment (rohanmathur91) [reply src/pages/Welcome/Welcome.jsx:27]: okay, nice.

**Review by NtshVrm — APPROVED** [review]
Good UI
Code is easy to read

**Pull request merged**